src/components/FileRepo/Stat.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FunctionComponent } from 'react';
+import { Spin } from 'antd';
+import QueriesResolver, {
+  Queries,
+  QueriesResolverFaCCType,
+} from '../CohortBuilder/QueriesResolver';
+import {
+  familyStatTag,
+  file_stat_tag,
+  fileSizeStatTag,
+  participantStatTag,
+} from './statsConstants';
+import { WarningTwoTone } from '@ant-design/icons';
+import './FileRepo.css';
+
+interface Props {
+  api: Function;
+  /* helps to debug in the network console */
+  tag:
```

Comment:
I think an enum here will make the code cleaner/clearer https://www.typescriptlang.org/docs/handbook/enums.html